SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLockingPolicy.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "SpatialConstants.h"
+
+#include "GameFramework/Actor.h"
+
+#include "AbstractLockingPolicy.generated.h"
+
+UCLASS(abstract)
+class SPATIALGDK_API UAbstractLockingPolicy : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	virtual ActorLockToken AcquireLock(AActor* Actor, FString LockName = "") PURE_VIRTUAL(UAbstractLockingPolicy::AcquireLock, return SpatialConstants::INVALID_ACTOR_LOCK_TOKEN;);
```

Comment:
Are you going to define-away the debug string (for example `#if !(UE_BUILD_SHIPPING || UE_BUILD_TEST)`)?

Reply:
Not planning to currently, zoning is pretty much debug-only for the next few months